dockers/docker-fpm-frr/supervisord.conf

```diff
@@ -21,55 +21,9 @@ startsecs=0
 stdout_logfile=syslog
 stderr_logfile=syslog
 
-[program:rsyslogd]
-command=/usr/sbin/rsyslogd -n
-priority=3
-autostart=false
-autorestart=false
-startsecs=0
-stdout_logfile=syslog
-stderr_logfile=syslog
-
-[program:zebra]
-command=/usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M fpm -M snmp
-priority=4
-autostart=false
-autorestart=false
-startsecs=0
-stdout_logfile=syslog
-stderr_logfile=syslog
-
-[program:staticd]
-command=/usr/lib/frr/staticd -A 127.0.0.1
-priority=4
-autostart=false
-autorestart=false
-startsecs=0
-stdout_logfile=syslog
-stderr_logfile=syslog
-
-[program:bgpd]
-command=/usr/lib/frr/bgpd -A 127.0.0.1 -M snmp
-priority=5
-stopsignal=KILL
-autostart=false
-autorestart=false
-startsecs=0
-stdout_logfile=syslog
-stderr_logfile=syslog
-
-[program:vtysh_b]
-command=/usr/bin/vtysh -b
```

Comment:
will vtysh_b started by itself?

Reply:
In daemons file `vtysh_enable=yes` will do that.